json_schemas/users.json

```diff
@@ -22,6 +22,11 @@
       "type": "string",
       "pattern": "^(buyer|supplier|admin)$"
     },
+    "supplier_id": {
+      "type": "string",
```

Comment:
supplier_id is an integer in the database, so probably makes sense to have a numeric field in the JSON?